src/components/Menu/index.tsx

```diff
@@ -143,21 +126,7 @@ const Menu: React.FC<{ chain?: number }> = () => {
         </Flex>
         {isMobile && <MobileMenu items={currentMenu(t) as any} />}
       </Box>
-      {!collapse && (
-        <Box
-          onClick={() => setCollapse(true)}
-          sx={{
-            display: ['none', 'block', 'block'],
-            position: 'fixed',
-            width: '100%',
-            height: '100%',
-            top: 0,
-            left: 0,
-            zIndex: 5,
-            background: 'rgb(56, 56, 56, 0.6)',
-          }}
-        />
-      )}
+      {!collapse && <Box onClick={() => setCollapse(true)} sx={styles.last} />}
```

Comment:
this is an overlay, not sure why the style is called last

Reply:
Changed name  to overlay